src/database/migrations/20200927093745-create-user.js

```diff
@@ -7,6 +7,16 @@ module.exports = {
         primaryKey: true,
         type: Sequelize.INTEGER
       },
+      roleId: {
+        allowNull: true,
+        type: Sequelize.INTEGER,
+        onDelete: 'CASCADE',
+        onUpdate: 'CASCADE',
+        references: {
+          model: 'UserRoles',
+          key: 'id'
+        }
+      },
```

Comment:
This migrations will not work, Remember that you are not supposed to edit old migrations, rather create a new one.

Reply:
What if I undo before migrate again I thought it can work

Reply:
No, create a new migration

Reply:
I did.